addons/Rakugo/lib/systems/Executer.gd

```diff
@@ -269,47 +273,8 @@ func do_execute_script(parameters:Dictionary):
 					else:
 						value = float(value)
 
-				var assignment = result["assignment"]
-				
-				var lvar_name = result["lvar_name"]
+				Rakugo.set_variable(result["lvar_name"], value)
```

Comment:
You gonna introduce regression here. Maybe copy/paste the code from main is the easiest way :)